ZividOpenCV/ZDF2OpenCV/ZDF2OpenCV.cpp

```diff
@@ -0,0 +1,172 @@
+/*
+Import a ZDF point cloud and convert it to OpenCV format.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/opencv.hpp>
+
+enum class Axis
+{
+    X,
+    Y,
+    Z
+};
+
+template<Axis c>
+static float getValue(const Zivid::Point &p);
+
+template<Axis c>
+static bool minNan(const Zivid::Point a, const Zivid::Point b);
+
+template<>
+static float getValue<Axis::X>(const Zivid::Point &p)
+{
+    return p.x;
+}
+
+template<>
+static float getValue<Axis::Y>(const Zivid::Point &p)
+{
+    return p.y;
+}
+
+template<>
+static float getValue<Axis::Z>(const Zivid::Point &p)
+{
+    return p.z;
+}
+template<Axis c>
```

Comment:
Nice how you use templates here! Then it will all be inlined when you compile with optimizations on.

Reply:
Bernhard helped me with that. I still don't understand the benefit.

Reply:
I can explain it irl later if you're interested?